src/main/webapp/js/lib/calendar.js

```diff
@@ -146,14 +181,16 @@ function addCourseToCalendar(course, startTime, endTime, day) {
  * @param {DateTime} startDate The DateTime object of the start time.
  * @param {DateTime} endDate The DateTime object of the end time.
  */
-function createSchedule(course, startDate, endDate) {
-  // TODO(savsa): Have each schedule be a different color. The color is
-  // currently hard coded to be blue so all schedules look the same.
+function createSchedule(course, startDate, endDate, color) {
+  // The default color is hard-coded to be blue;
+  if (color == null) {
+    color = '#00a9ff';
```

Comment:
this should be listed as a constant near the top of the file: `const DEFAULT_SCHEDULE_COLOR = '#00a9ff';`